examples/format-tag.c

```diff
@@ -52,7 +52,7 @@ int main(int argc, const char *argv[]) {
         MifareTag tag = tags[i];
 
         switch (freefare_get_tag_type(tag)) {
-            case DESFIRE:
+            case MIFARE_DESFIRE:
```

Comment:
Did you build `libfreefare` from master or from the latest release (0.4.0)? The symbols changed in master as part of a larger refactor but afaik that's still a WIP and things will not work if you try to run against it.

Reply:
Oh really? I did update it because libfreefare git has support for cmake...